api/python/provisioner/commands/deploy_vm.py

```diff
@@ -144,12 +158,16 @@ def _run_states(self, states_group: str, run_args: run_args_type):
                     self._apply_state(
                         f"components.{state}", secondaries, stages
                     )
-                elif state in ("ha.cortx-ha"):
-                    #Execute firt on primary, then on second node and then on the third
+                elif state in (
+                    "ha.cortx-ha"
+                ):
+                    # Execute first on primary then on secondary sequentially.
+                    nodes = DeployVM._get_node_list()
                     self._apply_state(f"components.{state}", primary, stages)
-                    self._apply_state(f"components.{state}", second_node, stages)
```

Comment:
Don't you think that we need similar changes in https://github.com/Seagate/cortx-prvsnr/blob/pre-cortx-1.0/api/python/provisioner/commands/deploy.py?

Reply:
comment is addressed